src/export.h

```diff
@@ -0,0 +1,29 @@
+#ifndef EXPORT_H_
+#define EXPORT_H_
+
+#include <vector>
+#include <span>
+#include <cstdint>
+
+using std::uint8_t;
+using std::uint32_t;
+
+struct Rgba
+{
+	uint8_t r, g, b, a;
+};
+
+struct Extent
+{
+	uint32_t w, h;
```

Comment:
Uninitialized garbage. `w` and `h` are not very obvious, either use `x` and `y` or `width` and `height`.